packages/ui/index.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render } from "@testing-library/react";
```

Comment:
1. Lets call our test files `*.test.js`... I think both `React` and `MUI` uses that naming. `*.spec.js` makes me think of requirements specification or something.
2. This is a `Button` component test so it should be in `Button.test.js`, no?

Reply:
@kilemensi Makes sense 👍🏽